hana/monitoring.sls

```diff
@@ -62,7 +62,7 @@ prometheus-hanadb_exporter:
 hanadb_exporter_configuration_{{ exporter_instance }}:
   file.managed:
     - source: salt://hana/templates/hanadb_exporter.j2
-    - name: /usr/etc/hanadb_exporter/{{ exporter_instance }}.json
+    - name: /etc/hanadb_exporter/{{ exporter_instance }}.json
```

Comment:
We cannot change this

Reply:
Okay, but it does not match to the documentation and the same is true for the logging-configuration.

Reply:
It is documented here: https://github.com/SUSE/hanadb_exporter#running-as-a-daemon

Better to keep the config files under `/usr/etc` to avoid issues with package updates that bring the configuration file